kratos/python/add_matrix_to_python.cpp

```diff
@@ -76,6 +76,22 @@ namespace Python
         //here we add the dense matrix
         auto matrix_binder = CreateMatrixInterface< DenseMatrix<double> >(m,"Matrix");
         matrix_binder.def(py::init<const DenseMatrix<double>::size_type, const DenseMatrix<double>::size_type>());
+        matrix_binder.def("__init__", [](DenseMatrix<double> &m, py::buffer b){
+          py::buffer_info info = b.request();
+          KRATOS_ERROR_IF( info.format != py::format_descriptor<double>::value ) << "Expected a double array\n";
+          KRATOS_ERROR_IF( info.ndim != 2 ) << "Incompatible buffer dimension\n";
+          m = DenseMatrix<double>(info.shape[0], info.shape[1]);
+
+          std::size_t count = 0;
+          for( int i=0; i<info.shape[0]; ++i )
```

Comment:
```suggestion
          for( int i=0; i<info.shape[0]; ++i ) {
```

should be on the same line

Reply:
is this a style convention? I saw it done in both ways and personally, I like the "more lines" version more. But I can of course change it.

Reply:
I am not sure if it was added to the style-guide, but I think this is used by most developers